script/fetch_and_build_all_images.sh

```diff
@@ -1,44 +1,49 @@
-TEST_DIR='_test_dir'
 
```

Comment:
I would add a shebang to this script that indicates that this should be run using bash:
```suggestion
#!/bin/bash
```